project.clj

```diff
@@ -1,4 +1,4 @@
-(defproject tvirolai/blurhash "0.0.8-SNAPSHOT"
+(defproject tvirolai/blurhash "_"
```

Comment:
These changes follow the instructions from https://github.com/esuomi/git-revisions-lein/blob/master/README.md

To summarize,
 - version string is changed to `"_"` to indicate it is being automatically generated by something else
 - `lein-git-revisions` is the plugin itself, `lein-pprint` is used in workflows
 - username+password based authentication instead of gpg. My hats off to you for apparently having had the ability to use it at some point, to me gpg is madness.
 - removed `:release-tasks` customization because that stuff is done elsewhere
 - added default configuration, which also outputs `resources/metadata.edn` Basically whenever the plugin is executed, it plops out that file which contains some metadata about the project's revision state. Mostly git values, stuff like this:
     ```clojure
     {:git/unversioned? false, 
      :revision "0.1.0-SNAPSHOT", 
      :git/dirty? true, 
      :git/branch "automated_version_bumping", 
      :git/tag "v0.0.0, origin/master, origin/HEAD, master", 
      :git/ref-short "7557795", 
      :git/ahead 0, 
      :git/ref "75577959e7c05415475464741b716945d1c9b281", 
      :git/tag-ref "75577959e7c05415475464741b716945d1c9b281", 
      :git/untagged? false, 
      :git/ahead? false, 
      :git/commits "80"}
     ```